io_scene_godot/converters/utils.py

```diff
@@ -1,7 +1,20 @@
 """Util functions and structs shared by multiple resource converters"""
+import bpy, bmesh, mathutils
 
-import bpy
-import bmesh
+gdenums = {
```

Comment:
I think you can simply make `PRIMITIVE_POINTS` a global variable,  make a dict may have some unecessary performance issue.

Reply:
you must be crazy bro. i don't want to read bullshit like this when i wake up.